vm-builder/main.go

```diff
@@ -291,7 +311,8 @@ func main() {
 		SpecBuild:       "",  // overridden below if spec != nil
 		SpecMerge:       "",  // overridden below if spec != nil
 		InittabCommands: nil, // overridden below if spec != nil
-		ShutdownHook:    "",  // overridden below if spec != nil
+		AgettyInitLine:  getAgettyInitLine(*targetArch),
+		ShutdownHook:    "", // overridden below if spec != nil
```

Comment:
super tiny nit: maybe let's move `AgettyInitLine` into a blog above, so it doesn't cause an unnecessary diff?

Reply:
done